pkg/config.go

```diff
@@ -60,6 +60,34 @@ func replaceStringSecret(val string, secretValues map[string][]byte) string {
 	})
 }
 
+func replaceServiceConfigSecret(data map[string]interface{}, secretValues map[string][]byte) map[string]interface{} {
```

Comment:
Is this change required to make replacement more accurate and avoid accidentally produce invalid YAML? 

The change looks good to me, just want to understand why it was required.

Reply:
That's right.
Currently, `argocd-notifications-secret` is not supported to set multiline value such as private key.
If the user set multiline value, invalid data format of yaml will be created.